Makefile

```diff
@@ -13,6 +13,9 @@ lint: flake8 pylint
 flake8:
 	flake8 .
 
+unit-test: lint
```

Comment:
I think it's better to have unit-test independent from lint, as one's not always want to test a "clean" code